azure-mgmt-batch/HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+3.0.0 (2017-02-17)
+++++++++++++++++++
+
+- Updated `BatchAccount` model - support for pool allocation in the users subscription.
+- Updated `BatchAccount` model - support for referencing an Azure Key Vault when allocation pools in users subscription.
```

Comment:
"allocating pools in the user's subscription" ?
"for accounts created with a pool allocation mode of UserSubscription" is another way I've seen in the Swagger spec.